Dockerfile

```diff
@@ -113,10 +113,23 @@ ENV SOPS_SHA_256=610fca9687d1326ef2e1a66699a740f5dbd5ac8130190275959da737ec52f09
 RUN echo "${SOPS_SHA_256}  /sops" | shasum -c
 RUN chmod +x /sops
 
+# ====
+# AWS Sam Deployer
+#
+# https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html
+FROM installer as samcli
+RUN apt-get update \
+  && apt-get install -y \
+     python3 \
+     python3-pip \
+  && rm -rf /var/lib/apt/lists/*
+RUN pip3 install aws-sam-cli \
+  && rm -rf /root/.cache/pip
+
 # =====
 # Deployer
 #
-FROM node:14
+FROM node:14-bullseye
```

Comment:
This is the main reason that the python packages weren't able to be copied straight over. There are OS dependencies that were incompatible between the installer and final image.